recipes/pcraster/postinstall.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
What is this file for exactly?

Reply:
It is plainly copying a few PCRaster dlls into the bin directory such that our applications can be started. It is on the ToDo list to integrate that into our CMake setup, but we are not yet there. Historically, we used another post-processing script to generate a package with dlls not in the bin directory. Hope this is ok fttb, we are open to suggestions.

Reply:
That is what I thought.  This is really bad to do in a post-install script, because it will seriously mess up the RPATHs that conda sets whenever an environment is activated.  You need to do all of your shared object copying in the build script.

Reply:
I was a bit surprised that it works with the postprocessing script ;-) I'll integrate a proper install into the CMake setup, while at it also resolve the two other hackish parts. That will take a while.
Thanks for your comments so far!

Reply:
My pleasure!